### PR TITLE
ramips-rt305x: remove opkg and USB packages from 4M flash boards

### PR DESCRIPTION
**Pull request by rotanid**
reduce size by removing dispensable packages,
to fix builds of this target in most cases.

result of discussion in #1782 

this PR has been build-tested

### CODE REVIEW

File: targets/ramips-rt305x [hunk 1]
```diff
@@ -1,4 +1,14 @@
 config '# CONFIG_KERNEL_KALLSYMS is not set'
+config 'CONFIG_GLUON_SPECIALIZE_KERNEL=y'
+
+no_opkg()
+
+packages {
+	'-kmod-usb-core',
+	'-kmod-usb2',
```
Comment (mweinelt) [inline targets/ramips-rt305x]: Are you sure rt305x has kmod-usb2?

Comment (NeoRaider) [reply targets/ramips-rt305x]: @mweinelt Many devices add kmod-usb2 in their DEVICE_PACKAGES.

**Comment by rotanid** [comment]
> If we apply the same limitations as ar71xx-tiny I'd also want to see this target marked as deprecated.

the really problematic devices are deprecated for a while now in master branch.
technically not all devices in this target need the limitations, but afaik you can only apply them per-target atm.
therefore, the affected devices could be removed for the next but one release and the restrictions for the target removed?

File: targets/ramips-rt305x (hunk 1 above)
Comment (rotanid) [reply targets/ramips-rt305x]: @mweinelt do you expect an answer from my side, too?
do you want me to "add" kmod-usb-dwc2 ?

Comment (rotanid) [reply targets/ramips-rt305x]: @mweinelt you were right, not kmod-usb2 has to be removed but kmod-usb-dwc2. i did this now and also changed the PR to only affect the 4mb devices.

**Pull request merged**